module/input/SensorFilter/src/SensorFilter.hpp

```diff
@@ -96,9 +96,31 @@ namespace module::input {
             } buttons;
 
             struct FootDown {
-                FootDown() : fromLoad(true), certaintyThreshold(0.05) {}
-                bool fromLoad;
-                float certaintyThreshold;
+                FootDown()
+                    : current_method("Z_HEIGHT")
```

Comment:
Please set these as defaults at the variable definitions, then make this `Footdown() = default;`